### ISSUE TITLE
[Accessibility]: Compare page: Diff has poor color contrast

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35536]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34426

### Problem description

<img width="1618" alt="image" src="https://user-images.githubusercontent.com/9516420/168788389-59b42852-4729-4b5d-a8d8-8d800faea916.png">

### Expected behavior

We should work with design to improve the diff colors to ensure better color contrast.

Note: Both for **dark** and **light** mode.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
The work to fix this has been merged in https://github.com/sourcegraph/sourcegraph/pull/37385. It is currently feature flagged so we can test internally before enabling it site-wide.

We have an issue (https://github.com/sourcegraph/sourcegraph/issues/37535) to enable it fully over the coming weeks, closing this in favour of that!
